docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -188,10 +235,9 @@ $ xtrabackup --prepare --target-dir=/data/backup \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
```

Comment:
are we talking about keyring component or keyring file here?

Reply:
then keyring_file_data is wrong option

Reply:
What option should we use instead?

Reply:
--component-keyring-config should be used for all components.